noir-projects/noir-contracts/contracts/token_with_refunds_contract/src/main.nr

```diff
@@ -425,10 +425,26 @@ contract TokenWithRefunds {
 
     // REFUNDS SPECIFIC FUNCTIONALITY FOLLOWS
     use dep::aztec::{
-        note::utils::compute_slotted_note_hiding_point_raw, prelude::FunctionSelector,
+        prelude::{FunctionSelector, NoteHeader},
         protocol_types::{storage::map::derive_storage_slot_in_map, point::Point}
     };
-
+    use crate::types::token_note::TokenNoteHidingPoint;
+
+    /// We need to use different randomness for the user and for the fee payer notes because if the randomness values
```

Comment:
Moved this over from the original description in `TokeNote` since it no longer made sense to have it there.